### ISSUE TITLE
Make this an alpha component

### ISSUE THREAD

**Issue by olivertzeng** [mimre25/jokes.nvim#2]

**Comment by mimre25** [comment]
Do you mean [alpha-nvim](https://github.com/goolord/alpha-nvim)?

If so, do you have a link to the documentation of how to do this?

I'm also taking contributions :slightly_smiling_face:

**Comment by olivertzeng** [comment]
> Do you mean [alpha-nvim](https://github.com/goolord/alpha-nvim)?
> 
> 
> 
> If so, do you have a link to the documentation of how to do this?
> 
> 
> 
> I'm also taking contributions :slightly_smiling_face: 

Yeah, seeing the jokes you wrote really want to have that on alpha-nvim!
